packages/sommelier.rb

```diff
@@ -70,6 +70,7 @@ def self.build
       system "echo '  echo \"sommelier process \$SOMM is running\"' >> stopsommelier"
       system "echo '  exit 1' >> stopsommelier"
       system "echo 'fi' >> stopsommelier"
+      system "echo 'set -a && source ~/.sommelier.env && set +a && initsommelier' > startsommelier"
```

Comment:
Probably should add this line at the top:
```
system "echo '#!/bin/bash' > startsommelier"
system "echo 'set -a && source ~/.sommelier.env && set +a && initsommelier' >> startsommelier"
```
Notice the second line appends to the file.

Reply:
Oh I forgot it, thanks for your kind reminder